_topics/en/transaction-bloom-filtering.md

```diff
@@ -0,0 +1,115 @@
+aliases:
+  - BIP37
+  - Bloom filters
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightweight Client Support
+  - Privacy Problems
+  - P2P Network Protocol
+  - Security Problems
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Transaction bloom filtering** is a method that allows lightweight
+  clients to limit the amount of transaction data they receive from full
+  nodes to only those transactions that affect their wallet (plus a
+  configurable amount of additional transactions to to generate
+  plausible denaibliity about which transactions belong to the client).
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  [Bloom filters][] provide the ability to create a compact filter that
+  is guarantee to match a specified string with a configurable rate of
+  false positive matches for other strings.  A lightweight client can
+  create a bloom filter for all of its wallet addresses, send that
+  filter to a node using the P2P protocol messages defined in BIP37, and
+  then request a special form of blocks (merkle blocks) from the node.
+
+  A merkle block, also defined in BIP37, will contain only transactions
+  matching the previously sent filter plus the block header and a
+  partial merkle branch connecting each matching transaction to the
+  merkle root in the block header.
+
+  Clients will also receive announcements of new unconfirmed transactions being
+  relayed by the node if they match the filter.
+
+  When BIP37 was popular, most lightweight clients that used it ran on
+  mobile devices with limited bandwidth and so chose low false positive
+  rates to minimize their bandwidth use.  This meant that they
+  essentially gave their list of addresses to any node they contacted.
+  It was expected that privacy-focused users could mitigate this privacy
+  loss by setting a higher false positive rate, but research suggests
+  that the rate needs to be quite high in order to provide plausible
+  deniability.  The problem is a consequence of the different types of
+  matches produced by the filter: a client wallet's own transactions are
+  guaranteed to be matched by the filter.  By comparison, matches made
+  for plausible deniability are a random selection from the set of
+  transactions being searched for matches.  Many of the client's
+  transactions will be related to each other or have characteristics in
+  common, but the randomly-selected transactions are unlikely to be
+  related and will have roughly the same distribution of characteristics
+  found in the overall transaction set.  So, unless the set of false
+  positives is very high, it's often easy for an attacker to distinguish
+  between the client's transactions and the false positives.
+
+  As an additional problem, BIP37 filtering is performed independently
```

Comment:
> ...for each client by nodes...

The 'by nodes' seems a bit disjointed in this sentence. Perhaps:

"As an additional problem, nodes serving  BIP37 filters must perform filtering independently for each client, and it's possible for ..."